DuckDuckGo/OmniBar.swift

```diff
@@ -162,6 +172,12 @@ class OmniBar: UIView {
         onQuerySubmitted()
     }
     
+    @objc private func reloadSpeechRecognizerAvailability() {
+        assert(Thread.isMainThread)
```

Comment:
do we need this assert here?

Reply:
Defensive programming =D
This call changes the UI so it needs to be called on the main thread, I like asserts because:
1 - states the intention. Someone reading it knows it should be done on the main thread
2 - Easier to catch bugs during development